src/sentry/receivers/onboarding.py

```diff
@@ -402,24 +410,25 @@ def record_event_with_first_minified_stack_trace_for_project(project, event, **k
         )
         return
 
-    # First, only enter this logic if we've never seen a minified stack trace before
-    if not project.flags.has_minified_stack_trace:
```

Comment:
Why are we removing this check?

Reply:
We already have a similar check before calling the signal

https://github.com/getsentry/sentry/blob/cc743eef0ac1aa8313e345fad998e2c2148512c2/src/sentry/event_manager.py#L613-L615

Reply:
this was being checked twice

Reply:
My concern here is that this is a separate function, and someone could call it without performing that check beforehand. If it was high volume, then that could result in a lot more queries to postgres.

Reply:
I understand your concern but I was not sure if would be better to add the code in the signal or here. What do you recommend? cc @scefali 

Would a comment help?

Reply:
I think the check here wasn't really hurting anything, it might be fine to just leave it in place?

Reply:
@priscilawebdev @wedamija The check for `if not project.flags.has_minified_stack_trace:` should still be there, it's only the date check `project.date_added > START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ` that should be moved.

Reply:
The goal is that we want to set the flag for older projects but not analytics so we don't send 100k analytics events for `first_event_with_minified_stack_trace_for_project.sent` that have been receiving stack traces for a while. Then after a few weeks we can remove the check for `START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ` because at that point only projects that _actually_ have new minified stack traces when they didn't before will trigger analytics.

Reply:
test added @scefali https://github.com/getsentry/sentry/blob/536eccf3741ba4ee1e75f69e817a6e35962120b7/tests/sentry/receivers/test_onboarding.py#L630